package.json

```diff
@@ -87,7 +87,15 @@
   "peerDependencies": {
     "redux-saga": "^1.0.1",
     "@redux-saga/is": "^1.0.1",
-    "@redux-saga/symbols": "1.0.1"
+    "@redux-saga/symbols": "^1.0.1"
+  },
+  "peerDependenciesMeta": {
+    "@redux-saga/is": {
+      "optional": true
```

Comment:
This is not optional, it's required by expectSaga.

Reply:
@noahnu Yes you are right. I fixed it.